### PR TITLE
Surgeons

### PR DESCRIPTION
**Pull request by MarkSuckerberg**
## About The Pull Request
Was mostly coded by @LastCrusader105 , and it has been tested thoroughly. It seems like a cool idea.

## Why It's Good For The Game
It would allow someone whose job is actually to do surgery, an often neglected mechanic. It usually is handled by MDs, but I think having one person completely focused on doing it would lead to more use of the feature.

## Changelog
:cl: LastCrusader105
add: Surgeons as a new job type
/:cl:

### CODE REVIEW

**Comment by MarkSuckerberg** [comment]
This is why I hate mapmerge.

**Comment by Crossedfall** [comment]
You should setup the auto-mapmerge git hook. It will make your life 1000% easier.

**Comment by MarkSuckerberg** [comment]
It always ends up breaking, I can try getting it again though

File: code/modules/jobs/access.dm [hunk 1]
```diff
@@ -347,7 +347,7 @@
 	return list("Assistant", "Captain", "Head of Personnel", "Bartender", "Cook", "Botanist", "Quartermaster", "Cargo Technician",
 				"Shaft Miner", "Clown", "Mime", "Janitor", "Curator", "Lawyer", "Chaplain", "Chief Engineer", "Station Engineer",
 				"Atmospheric Technician", "Chief Medical Officer", "Medical Doctor", "Chemist", "Geneticist", "Virologist",
-				"Research Director", "Scientist", "Roboticist", "Head of Security", "Warden", "Detective", "Security Officer")
+				"Research Director", "Scientist", "Roboticist", "Head of Security", "Warden", "Detective", "Security Officer", "Paramedic", "Surgeon")
```
Comment (qwertyquerty) [inline code/modules/jobs/access.dm:350]: Paramedic?

File: code/modules/jobs/job_types/emt.dm [hunk 2]
```diff
@@ -16,7 +16,7 @@
 	paycheck = PAYCHECK_MEDIUM
 	paycheck_department = ACCOUNT_MED
 
-	display_order = JOB_DISPLAY_ORDER_MEDICAL_DOCTOR
+	display_order = JOB_DISPLAY_ORDER_EMT
```
Comment (qwertyquerty) [inline code/modules/jobs/job_types/emt.dm:19]: EMT?

**Comment by qwertyquerty** [comment]
Looks like you slipped some things related to EMTs in here.

**Comment by LastCrusader105** [comment]
Nice work guys

**Comment by MarkSuckerberg** [comment]
Yeah just fixes, probably should have stashed those and made another PR, can remove them if ya want

File: code/modules/jobs/access.dm (hunk 1 above)
Comment (ike709) [reply code/modules/jobs/access.dm:350]: Looks like he forgot to add it when he added EMTs.

File: code/modules/jobs/job_types/emt.dm (hunk 2 above)
Comment (ike709) [reply code/modules/jobs/job_types/emt.dm:19]: Looks like another oversight being fixed from the EMT PR.

**Review by ike709 — CHANGES_REQUESTED** [review]
I think the number of slots should be increased to at least 2-3 and surgery access should potentially be removed from medical doctors.

**Comment by jlsnow301** [comment]
On the same mindset with @ike709 as I can see this easily going awry. I think a crew objective of "first do no harm" would be nice (by not killing a player with the easy access bone saw)

**Comment by MarkSuckerberg** [comment]
Alright, maps updated, job slots and access updated, anything I'm missing?

**Comment by qwertyquerty** [comment]
I disagree with this. A surgeon is way too much of a niche job to have like 2-3 spots. Adding surgery access to MDs also makes them useless. Decreasing the slot amount makes no surgery ever get done. Either this should just be an alternate job title (which need porting) or needs to be rethought a bit more.

**Comment by MarkSuckerberg** [comment]
> I think the number of slots should be increased to at least 2-3 and surgery access should potentially be removed from medical doctors.

I was just going off of Ike's suggestion. It was originally only one slot, and MDs still had surgery access.

**Comment by ike709** [comment]
I went on a discord rant about how we should port alt job titles and link it to outfits rather than bloating the job list.

**Comment by LastCrusader105** [comment]
All my work for nothing
What if I add spare eyes and stuff to the surgery then it can be used to fix people who take the blind trait etc?

**Comment by Toxici11i** [comment]
> adding more jobs to medical
> said jobs have no relevancy since most of medical can still be cured very quickly via cryo showers or chemicals
> also the fact the tend wounds PR was never merged

**Comment by ike709** [comment]
Job panel bloat < alt job title system

**Pull request closed without merging**